baksmali/src/main/java/org/jf/baksmali/Adaptors/Format/InstructionMethodItem.java

```diff
@@ -394,95 +394,95 @@ protected void writeThirdRegister(IndentingWriter writer) throws IOException {
     }
 
     protected void writeInvokeRegisters(IndentingWriter writer) throws IOException {
-        FiveRegisterInstruction instruction = (FiveRegisterInstruction)this.instruction;
```

Comment:
The changes in this class are too spammy. I don't think there is any chance for confusion here, because the field is final and the local variables contain the same value as the field, just using a more specific type.